Lib/glyphsLib/classes.py

```diff
@@ -1612,11 +1612,13 @@ class GSNode(GSBase):
     )
     _parent = None
 
-    def __init__(self, position=(0, 0), nodetype=LINE, smooth=False, name=None):
+    def __init__(self, position=(0, 0), type=LINE, smooth=False, name=None, nodetype=None):
```

Comment:
Why was it necessary to also rename the nodetype parameter?

Reply:
I just saw your comment, thanks

Reply:
My thought process went: the actual constructor interface should be called `type`, to match Glyphs; but we should also support `nodetype` in case anyone's relying on `glyphsLib` having that interface.

Reply:
that's fine. I was suggesting that we also do a `warnings.warn("nodetype is deprecated, please use type instead", DeprecationWarning)` and in a few releases we remove it altogether